### PR TITLE
Update 2-ui/4-forms-controls/1-form-elements/article.md

### PR DESCRIPTION
**Pull request by dorelljames**
This updates a small grammar error and adds some helpful info. Hoping it's better now. Let me know if you need any more changes and I'll update it immediately.

### CODE REVIEW

File: 2-ui/4-forms-controls/1-form-elements/article.md [hunk 1]
```diff
@@ -205,7 +205,9 @@ Here is an example:
 </script>
 ```
 
-Unlike most other controls, `<select>` allows to select multiple options at once if it has `multiple` attribute. That's feature is rarely used. In that case we need to use the first way: add/remove the `selected` property from `<option>` subelements.
+Unlike most other controls, `<select>` allows to select multiple options at once if it has `multiple` attribute. Although such functionality is available, it is rarely used because of the different ways of doing the selection which vary in different operating systems and browser. And also, because you have to inform the user that multiple selection is available, it is more user-friendly to use checkboxes instead.
```
Comment (iliakan) [inline 2-ui/4-forms-controls/1-form-elements/article.md:208]: Unlike most other controls, `<select>` allows to select multiple options at once if it has `multiple` attribute. Although such functionality is available, it is rarely used.

**Review by iliakan — CHANGES_REQUESTED** [review]
Make it shorter? More words don't put it clear, I'm afraid.

File: 2-ui/4-forms-controls/1-form-elements/article.md (hunk 1 above)
Comment (dorelljames) [reply 2-ui/4-forms-controls/1-form-elements/article.md:208]: Got it.

**Comment by dorelljames** [comment]
I've made the necessary changes. Thanks

**Comment by iliakan** [comment]
Thank you.

**Pull request merged**